Undocumented dependencies

The `libnuma` dependency is not documented in [INSTALL.md](https://github.com/ginkgo-project/ginkgo/blob/develop/INSTALL.md).

```
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find NUMA (missing: NUMA_LIBRARY NUMA_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindNUMA.cmake:41 (find_package_handle_standard_args)
  cuda/test/base/CMakeLists.txt:6 (find_package)


-- Configuring incomplete, errors occurred!
```

`doxygen` and (optionally) `dot`/`graphviz` are required for building the documentation.

I also noticed `No OpenCV found, disabling examples with video output` so that should be documented too.

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -219,7 +219,7 @@ packages can be turned off by disabling the relevant options.
   [OpenCV](https://opencv.org/) is required for some examples, they are disabled when OpenCV is not available.
 + GINKGO_BUILD_DOC=ON:
   [doxygen](https://www.doxygen.nl/) is required to build the documentation and
-  additionally [graphviz](https://graphviz.org/) are required to build the class hierarchy graphs.
+  additionally [graphviz](https://graphviz.org/) is required to build the class hierarchy graphs.
 
 Ginkgo attempts to use pre-installed versions of these package if they match
 version requirements using `find_package`. Otherwise, the configuration step

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -212,6 +212,14 @@ packages can be turned off by disabling the relevant options.
 + GINKGO_BUILD_HWLOC=ON:
   [hwloc](https://www.open-mpi.org/projects/hwloc) to detect and control cores
   and devices.
++ GINKGO_BUILD_HWLOC=ON and GINKGO_BUILD_TESTS=ON:
+  [libnuma](https://www.man7.org/linux/man-pages/man3/numa.3.html) is required
+  when testing the functions provided through MachineTopology.
++ GINKGO_BUILD_EXAMPLES=ON:
+  [OpenCV](https://opencv.org/) is required for some examples.
++ GINKGO_BUILD_DOC=ON:
+  [doxygen](https://www.doxygen.nl/) is required to build the documentation and
+  additionally graphviz and dot are required to build the class hierarchy graphs.
 
 Ginkgo attempts to use pre-installed versions of these package if they match
 version requirements using `find_package`. Otherwise, the configuration step

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -216,10 +216,10 @@ packages can be turned off by disabling the relevant options.
   [libnuma](https://www.man7.org/linux/man-pages/man3/numa.3.html) is required
   when testing the functions provided through MachineTopology.
 + GINKGO_BUILD_EXAMPLES=ON:
-  [OpenCV](https://opencv.org/) is required for some examples.
+  [OpenCV](https://opencv.org/) is required for some examples, they are disabled when OpenCV is not available.
 + GINKGO_BUILD_DOC=ON:
   [doxygen](https://www.doxygen.nl/) is required to build the documentation and
-  additionally graphviz and dot are required to build the class hierarchy graphs.
+  additionally [graphviz](https://graphviz.org/) are required to build the class hierarchy graphs.
 
 Ginkgo attempts to use pre-installed versions of these package if they match
 version requirements using `find_package`. Otherwise, the configuration step

--- a/include/ginkgo/core/stop/residual_norm.hpp
+++ b/include/ginkgo/core/stop/residual_norm.hpp
@@ -364,7 +364,7 @@ protected:
 /**
  * The AbsoluteResidualNorm class is a stopping criterion which stops the
  * iteration process when the residual norm is below a certain
- * threshold, i.e. when norm(residual) / threshold.
+ * threshold, i.e. when norm(residual) < threshold.
  * For better performance, the checks are run thanks to kernels on
  * the executor where the algorithm is executed.
  *

